src/commands/scanner/run/dfa.ts

```diff
@@ -95,6 +96,12 @@ export default class Dfa extends ScannerRunCommand {
 			longDescription: messages.getMessage('flags.ignoreparseerrorsDescriptionLong'),
 			env: 'SFGE_IGNORE_PARSE_ERRORS'
 		}),
+		// NOTE: This flag can't use the `env` property to inherit a value automatically, because OCLIF boolean flags
+		// don't support that. Instead, we check the env-var manually in a subsequent method.
+		'rule-disable-warning-violation': flags.boolean({
```

Comment:
I renamed the flag to `...disable...` instead of `...enable`, because using `...enable` would have required a boolean flag whose default value is true, and having the user make it false by doing `--no-...`, which seems unnecessarily cumbersome. This way, they can just supply the flag to deviate from the default behavior, instead of needing `--no-`.

Reply:
Makes sense.